config/resources/besluit-domain.lisp

```diff
@@ -10,7 +10,7 @@
                               :as "created-for")
              (agendastatus    :via        ,(s-prefix "besluitvorming:agendaStatus")
                               :as "status")
-             (access-level    :via ,(s-prefix "besluitvorming:vertrouwelijkheidsniveau")
+             (concept    :via ,(s-prefix "besluitvorming:vertrouwelijkheidsniveau")
```

Comment:
This relation looks like legacy and can just be removed? 
I cannot find a triple using this predicate in the production DB.